module/data/actor/templates/traits.mjs

```diff
@@ -124,6 +124,8 @@ export default class TraitsField {
   static prepareResistImmune() {
     if ( this.parent.hasConditionEffect("petrification") ) {
       this.traits.dr.custom = game.i18n.localize("DND5E.DamageAll");
+      Object.keys(CONFIG.DND5E.damageTypes).forEach(type => this.traits.dr.value.add(type));
```

Comment:
I think this results in spamming the character sheet with a huge amount of damage immunity pills, which is why we opted for the 'All' custom damage immunity instead. It might be preferable to defer this until we can add special-case handling for an 'all' type in the system as part of the resistances/immunities/vulnerability automation.

It's not very often that you are petrified though so I don't feel strongly.

Reply:
I added special handling for petrification to the Actor sheet to avoid the spam.